pkg/command/service/clean.go

```diff
@@ -0,0 +1,139 @@
+// Copyright © 2020 The Knative Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package service
+
+import (
+	"fmt"
+	"os"
+	"strconv"
+	"strings"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	_ "k8s.io/client-go/plugin/pkg/client/auth/oidc"
+
+	servingv1client "knative.dev/serving/pkg/client/clientset/versioned/typed/serving/v1"
+
+	"github.com/spf13/cobra"
+
+	"knative.dev/kperf/pkg"
+	"knative.dev/kperf/pkg/generator"
+)
+
+func NewServiceCleanCommand(p *pkg.PerfParams) *cobra.Command {
+	ksvcCleanCommand := &cobra.Command{
+		Use:   "clean",
+		Short: "clean ksvc",
+		Long: `clean ksvc workload
+
+For example:
+# To clean ksvc workload
+kperf service clean --nsprefix testns/ --ns nsname
+`,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			var nsRangeMap map[string]bool = map[string]bool{}
+			if nsPrefix != "" {
+				r := strings.Split(nsRange, ",")
+				if len(r) != 2 {
+					fmt.Printf("Expected Range like 1,500, given %s\n", nsRange)
+					os.Exit(1)
+				}
+				start, _ := strconv.Atoi(r[0])
+				end, _ := strconv.Atoi(r[1])
```

Comment:
don't ignore errors.
Also you can use `fmt.Sscanf` to read the whole pair.

Reply:
@vagababov Thanks! Will fix it in next patch.